DonoGear/app/src/main/java/com/example/donogear/LoginPageActivity.java

```diff
@@ -61,11 +84,69 @@ public void onTabReselected(TabLayout.Tab tab) {
         });
     }
 
+    private void alertDisplayer(String title,String message){
+        AlertDialog.Builder builder = new AlertDialog.Builder(LoginPageActivity.this)
+                .setTitle(title)
+                .setMessage(message)
+                .setPositiveButton("OK", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialog, int which) {
+                        dialog.cancel();
+//                        Intent intent = new Intent(LoginPageActivity.this, MainActivity.class);
+//                        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK | Intent.FLAG_ACTIVITY_NEW_TASK);
+//                        startActivity(intent);
+                    }
+                });
+        AlertDialog ok = builder.create();
+        ok.show();
+    }
+
     public void createAccountBtnClicked(View view) {
 
+        username = usernameEditTextSignup.getText().toString();
```

Comment:
Need to add a null check for the user name and the password. Don't want the user keeping these fields blank